docs/src/main/sphinx/admin/spill.md

```diff
@@ -12,6 +12,12 @@ implemented on the application level to address specific needs of Trino.
 
 Properties related to spilling are described in {doc}`properties-spilling`.
 
+:::{warning} 
+The spill to disk feature and implementation are a legacy functionality of
+Trino. Consider using [](/admin/fault-tolerant-execution) with the `task` retry
```

Comment:
FTE is not really a replacement but solves the memory issue that spilling attempts to solve.